aggregator/pom.xml

```diff
@@ -22,12 +22,12 @@
     <parent>
```

Comment:
2022 copyrights here and in many other files

Reply:
thx! will update them all